src/Fees.sol

```diff
@@ -1,58 +1,66 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.20;
 
+import {Log2} from "./Log2.sol";
+
 library PDPFees {
+    uint256 constant ATTO_FIL = 1;
+    uint256 constant ONE_FIL = 1e18 * ATTO_FIL;
 
-    // Estimate of USD price in FIL
-    uint256 constant FIL_USD_NUM = 10;
-    uint256 constant FIL_USD_DENOM = 35;
-           
-    uint256 constant ONE_PERCENT_DENOM = 100;
-
-    uint256 constant LEAVES_PER_TIB = 2^35;
-
-    uint256 constant EPOCHS_PER_MONTH = 86_400;
-    // Revenue in usd per TiB per month
-    uint256 constant MONTHLY_TIB_REVENUE_USD_NUM = 35;
-    uint256 constant MONTHLY_TIB_REVENUE_USD_DENOM = 10;
-
-    uint256 constant PROOF_GAS_FLOOR = 2_000_000;
-    uint256 constant ONE_NANO_FIL = 10^9;
-    uint256 constant ONE_FIL = 10^18;
-
-    uint256 constant SYBIL_FEE = 10^17;
-
-    // Currently unused
-    //
-    // Returns the service fee for a given proofset duration, and size
-    // Size measured in leaves (32 byte chunks), duration measured in epochs
-    function serviceFee(uint256 duration, uint256 size) internal pure returns (uint256) {
-        // 1 FIL / TiB / month ~ revenue 
-        // 86,400 epochs / month
-        // 2^35 leafs / TiB
-        // fee is 1% of revenue
-
-        // [(Monthly revenue in USD) * (FIL / USD) * size * duration] / 
-        // [Leaves per TiB * Epochs per month * 1%]
-        uint256 numerator = ONE_FIL * MONTHLY_TIB_REVENUE_USD_NUM * FIL_USD_NUM * size * duration;
-        uint256 denominator = MONTHLY_TIB_REVENUE_USD_DENOM * FIL_USD_DENOM * LEAVES_PER_TIB * EPOCHS_PER_MONTH * ONE_PERCENT_DENOM;
-        return numerator / denominator;
-    }
+    // 0.1 FIL
+    uint256 constant SYBIL_FEE = ONE_FIL / 10;
+
+    // assume 1FIL = $5 for now -> we can change this to use an oracle in PDP V1
+    uint256 constant FIL_USD_PRICE = 5;
+
+    // 2 USD/Tib/month is the current reward earned by Storage Providers
+    uint256 constant MONTHLY_TIB_STORAGE_REWARD_USD = 2;
+
+    uint256 constant DAILY_TIB_STORAGE_REWARD_ATTO_FIL =
+        (MONTHLY_TIB_STORAGE_REWARD_USD * 1e18 * ATTO_FIL) /
+            (30 * FIL_USD_PRICE);
+
+    // PROOF_PRICE is currently set to 1% of the daily reward
+    uint256 constant PROOF_PRICE_ATTO_FIL =
+        (1 * DAILY_TIB_STORAGE_REWARD_ATTO_FIL) / 100;
```

Comment:
Note that this is aligned 100% with what happens in simple PDP service. 
Just want to point out that if anyone deploys his own PDPService contract (with different frequency) er have two options
- make clear that in this case one should also update the proofFee definition accordingly
- define proofFee in terms of MaxProvingPeriod (i.e. Proof_Price_Atto_Fil = x*epochReward*getMaxProvingPeriod())

If this does not happen, in case of changing the proof frequency the SP is paying the 1% of the daily reward each time he posts a proof. 

Note that in the second suggested option we have as a prerequisite the access to getMaxProvingPeriod(), which "rules out" policies which are not based on proving with some frequency (for instance: prove upon client request, no fixed schedule).

Reply:
@aarshkshah1992 I think what we call Proof_price here is actually ProofFee in the doc https://www.notion.so/filecoindev/Pricing-mechanism-for-PDPverifier-12adc41950c180ea9608cb419c369ba4

Reply:
See comment on Proof_Price below. the formula here is the formula of ProofFee wrt requirement 2 here 
https://www.notion.so/filecoindev/Pricing-mechanism-for-PDPverifier-12adc41950c180ea9608cb419c369ba4?pvs=4#13adc41950c180709d5ed5142aee845d

Reply:
@lucaniz Yeah renamed this variable to "proof fee" which is 1% of DAILY REWARD.

Reply:
@lucaniz @irenegia I am a bit confused on the right naming. Is 1% of the daily reward the "proof fee" or the "proof price" ? To me, "price" is more intuitive like price of 1 banana and then to get the "total proof fee" I scale it by proof size. Lemme know what you'll think.

Reply:
I think "proof price" and "proof cost" is a good naming convention here. But will defer to you'll.

Reply:
The original comment is now out of date given that we are measuring proving time between calls successive proof calls..